<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>license_button</class>
 <widget class="QDialog" name="license_button">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>61</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_license">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>421</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Afficher">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_quit">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>340</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>341</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Liste des licenses</string>
     </property>
     <widget class="QPushButton" name="pushButton_sup">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>60</y>
        <width>75</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_license">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>401</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11_license">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN a supprimer:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CINsup_license">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CINrec_license">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rec_license">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>150</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4_license">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Entrer CIN a rechercher:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="export_excel">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Ajouter">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Ajouter/Modifier</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter une license</string>
     </property>
     <widget class="QLabel" name="label_license">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2_license">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3_license">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Ajouter_license">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CIN_license">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nom_license">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Pr_license">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6_license">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom du licenses:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nl_license">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14_license">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mail_license">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_mod_license">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>170</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Email">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Envoi E-mail</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ã€:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Sujet:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Attachement">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Attachement:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rcpt">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="file">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>441</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>190</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mail_pass">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Votre mot de passe:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browseBtn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Parcour...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
